ush/fv3gfs_remap.sh

```diff
@@ -5,16 +5,13 @@
 #-- Fanglin Yang, October 2016
```

Comment:
I don't think we use this script anymore.

Reply:
You're right, we don't. I just searched for it within the system and did not find any. I will remove this script.

Reply:
Script has been sent to the farm @ e8136b60